Automattic/woocommerce-services
# Transform order object to a presentable API object

Closes https://github.com/Automattic/woocommerce-services/issues/2116

### Description
WCS 1.24.0 [initializes order](https://github.com/Automattic/woocommerce-services/blob/d7f915ed13cb53dea0118325da0d912e93b8d501/classes/class-wc-connect-shipping-label.php#L420) as the initial states of the react app. 

Currently, `$order->get_data()` returns a PHP object of the order. This object is not suitable to output as `json_encode()`, some of its protected fields are not generated, resulting a `{}` at the end. 

We should transform this order object to something that can be represented as a JSON object. This can be done similar to [Woo core's API](https://github.com/woocommerce/woocommerce/blob/cbfe4bd59587aadb4b6c6e95a5787bebe3925310/includes/legacy/api/v3/class-wc-api-orders.php#L142-L320). 

### To test
1. Go to an order page, and click to the step where you can see rates.
2. Verify the "Customer paid..." box shows
![image](https://user-images.githubusercontent.com/572862/89829471-e28f6300-db17-11ea-8aea-841dbbc15454.png)
